plugins/GSdx/GSTextureCache.cpp

```diff
@@ -48,7 +48,9 @@ GSTextureCache::GSTextureCache(GSRenderer* r)
 
 	m_paltex = theApp.GetConfigB("paltex");
 	m_can_convert_depth &= s_IS_OPENGL; // only supported by openGL so far
-	m_crc_hack_level = theApp.GetConfigI("crc_hack_level");
+	m_crc_hack_level = theApp.GetConfigT<CRCHackLevel>("crc_hack_level");
+	if (m_crc_hack_level == CRCHackLevel::Automatic)
+		m_crc_hack_level = AutoSelectCRCHackLevel(theApp.GetConfigT<GSRendererType>("Renderer"));
```

Comment:
This might not work properly on Windows. If the renderer type is `GSRendererType::Default`, then you may end up using an OpenGL renderer but have CRC hacks at the full level due to the renderer autoselection code. The same applies for the other uses of `AutoSelectCRCHackLevel()`.

One solution would be to check if `m_crc_hack_level` is set to `CRCHackLevel::Automatic` in the most derived constructor (GSTextureOGL, GSRendererDX11, etc.) and adjust the level there if it is. There may be other/better solutions.

(Unrelated to PR): I note that s_IS_OPENGL might not work properly on Windows for the same reasons.